src/schema/resolvers/user-resolver.js

```diff
@@ -0,0 +1,12 @@
+import * as user from '../../controllers/user-controller.js'
+
+export default {
+  Query: {
+    users: (parent, args, { db }) => db.user.findAll(),
```

Comment:
Here, you're doing two things:
1. Creating a mapping: "this field should be resolved with this function"
2. You're also defining the function.

The function's body should be written under the controllers folder. Again, concepts are everything!